cmd/kubeapps-apis/plugins/helm/packages/v1alpha1/server_test.go

```diff
@@ -366,17 +471,56 @@ func TestGetAvailablePackageSummaries(t *testing.T) {
 					Cluster:   "",
 					Namespace: globalPackagingNamespace,
 				},
-				FilterOptions: &corev1.FilterOptions{
-					Query:        "",
-					AppVersion:   "",
-					PkgVersion:   "",
-					Categories:   nil,
-					Repositories: nil,
```

Comment:
Not sure why the FilterOptions are being specified here if we're not testing them at all (yet).

Reply:
I just tried to initialize every field for having a clear picture of what this model is comprised of. Then I deleted those unused fields... but not this one it seems. It's just a leftover, thanks for removing it!